resmaswrapped/OLD wrapped.py

```diff
@@ -0,0 +1,47 @@
+pip install spotipy
+import requests
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+import pandas as pd
+import time
+
+SPOTIPY_CLIENT_ID='insert client id'
+SPOTIPY_CLIENT_SECRET='insert client secret'
+SPOTIPY_REDIRECT_URI='http://localhost:8888/callback'
+SCOPE = 'user-top-read'
```

Comment:
It looks like the user needs to write this up manually, would it be easier to have them input this into a funtion?